# step, square, sinusoidal
signal_type : square
frequency : .25
amplitude: 20